src/main/java/de/tum/in/www1/artemis/service/connectors/AtheneService.java

```diff
@@ -0,0 +1,269 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import static de.tum.in.www1.artemis.config.Constants.ATHENE_RESULT_API_PATH;
+import static de.tum.in.www1.artemis.service.connectors.RemoteArtemisServiceConnector.authorizationHeaderForSymmetricSecret;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
+
+import java.util.*;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.enumeration.Language;
+import de.tum.in.www1.artemis.exception.NetworkingError;
+import de.tum.in.www1.artemis.repository.TextBlockRepository;
+import de.tum.in.www1.artemis.repository.TextClusterRepository;
+import de.tum.in.www1.artemis.repository.TextExerciseRepository;
+import de.tum.in.www1.artemis.service.TextAssessmentQueueService;
+import de.tum.in.www1.artemis.service.TextBlockService;
+import de.tum.in.www1.artemis.service.TextSubmissionService;
+import de.tum.in.www1.artemis.web.rest.dto.AtheneDTO;
+
+@Service
+@Profile("athene")
+public class AtheneService {
+
+    private final Logger log = LoggerFactory.getLogger(AtheneService.class);
+
+    @Value("${server.url}")
+    protected String ARTEMIS_SERVER_URL;
```

Comment:
we are now using camelCase for those variables.
Why is this one protected and not private?

Reply:
```suggestion
    protected String artemisServerUrl;
```